app/controllers/PostTravelProductController.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const db = require('../../config/sequelize');
+const StandardError = require('standard-error');
+
+exports.all = function(req, res){
+    db.post_travel_product.findAll({include: [{model:db.post_travel_product_document, attributes: ['image_name', 'image_path']}]}).then(
+        function(product){
+            return res.jsonp(product);
+        }).catch(function(err){
+            return res.jsonp(err);
+        });
+};
+
+exports.show = function (req, res) {
+    return res.jsonp(req.product);
+};
+
+exports.byId = function (req, res, next, id) {
+    console.log('id => ' + id);
+    db.post_travel_product.find({
+        where: { id: id },
+        include: [{
+            model: db.post_travel_product_document
+        }]
+    }).then(function (product) {
+        if (!product) {
+            return next(new Error('Failed to load postTraveProductlId ' + id));
+        } else {
+            req.product = product;
+            return next();
+        }
+    }).catch(function (err) {
+        return next(err);
+    });
+};
+
+exports.byProductCategoryId = function(req, res, next, id){
+    db.post_travel_product.findAll({
+        where: {product_category_id: id}
+
+    }).then(function(product){
+        req.product = product;
+        return next();
+        
+    }).catch(function(err){
+        return res.render('error', {
+            error: err,
+            status: 500
+        })
+    });
+}
+
+exports.byProfileId = function(req, res, next, id){
```

Comment:
`exports.byProfileId = function(req, res, next, id){`

Please put proper API naming.
`exports.byProfileId` 
Change to `exports.getPostTravelProductByProfileId`

Reply:
Updated the naming convention